AEPCampaign/Sources/CampaignState.swift

```diff
@@ -159,7 +158,10 @@ class CampaignState {
         return true
     }
 
-    ///Determines if the registration request should send to Campaign. Returns true, if the ecid has changed or number of days passed since the last registration is greater than registrationDelay in obtained in Configuration shared state.
+    /// Determines if a registration request should be sent to Campaign. Returns true if the ECID has changed or if the registrationDelay specified in the Configuration shared state has elapsed.
```

Comment:
Optional: When you want to provide additional details for a method, move it after method Summary (provide a blank line between the two). This way the method Summary remains brief and additional info appears in the Discussion section.